blueprints/app/files/.ember-cli

```diff
@@ -5,5 +5,11 @@
 
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": false
+  "disableAnalytics": false,
+
+  /**
+  Setting `isTypeScriptProject` to true will force the blueprint generators to generate TypeScript
+  rather than JavaScript by default
```

Comment:
```suggestion
  Setting `isTypeScriptProject` to true will force the blueprint generators to generate TypeScript
  rather than JavaScript by default, when a TypeScript version of a given blueprint is available.
```